src/site/content/en/blog/how-to-use-local-https/index.md

```diff
@@ -0,0 +1,317 @@
+
+<figure class="w-figure">
+  <img src="./selfSigned.jpg" alt="Why browsers don't trust self-signed certificates: diagram">
+  <figcaption class="w-figcaption">If you open your locally running site in your browser using HTTPS, your browser will check the certificate of your local development server. Upon seeing that the certificate has been signed by yourself, the browser checks whether you're registered as a trusted certificate authority. Because you're not, your browser can't trust the certificate; it displays a warning telling you that your connection is not secure. You may proceed at your own risk — if you do, an HTTPS connection will be created.</figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./warnings.jpg" alt="Screenshots of warnings showed by browsers when a self-signed certificate is used">
+    <figcaption class="w-figcaption">Warnings showed by browsers when a self-signed certificate is used.</figcaption>
+</figure>
+</figure>
+
+{% endDetails %}
+
+### Certificate signed by a regular certificate authority
+
+You may also find techniques based on having an actual certificate authority—not a local one—sign your certificate.
+
+A few things to keep in mind if you're considering using these techniques:
+
+- You'll have more setup work to do than when using a local CA technique like mkcert;
+- You need to use a domain name that you control and that is valid. This means that you **can't** use actual certificate authorities for:
+  - `localhost` and other domain names that are [reserved](https://www.iana.org/assignments/special-use-domain-names/special-use-domain-names.xhtml), such as `example` or `test`.
+  - Any domain name that you don't control.
+  - Top-level domains that are not valid. See the [list of valid top-level domains](https://www.iana.org/domains/root/db).
+
+### Reverse proxy
+
+Another option to access a locally running site with HTTPS is to use a [reverse proxy](https://en.wikipedia.org/wiki/Reverse_proxy) such as [ngrok](https://ngrok.com/).
+
+A few points to consider:
+
+- Anyone can access your local development site once you share with them a URL created with a reverse proxy. This can be very handy when demoing your project to clients. Or this can be a downside, if your project is sensitive.
+- You may need to consider pricing.
+- New [security measures](/cors-rfc1918-feedback/) in browsers may affect the way these tools work.
+
+### Flag
+
+If you're using a custom hostname like `mysite.example`, you can use a flag in Chrome to forcefully consider `mysite.example` secure. Avoid doing this, because:
+
+- You would need to be 100% sure that `mysite.example` always resolves to a local address, otherwise you could leak production credentials.
+- You won't be able to debug across browsers with this trick 🙀.
+
+_With many thanks for contributions and feedback to all reviewers and contributors—especially Ryan Sleevi,
+Filippo Valsorda, Milica Mihajlija, Rowan Merewood and Jake Archibald. 🙌_
```

Comment:
```suggestion
```

Reply:
I would prefer to keep these shoutouts in for now, unless it's a hard guideline!

Reply:
It's not, it's still under discussion and we'll be considering your perspective too. I'll ping you to get some feedback on this later if that's okay :)

Reply:
Sounds good, thanks!